RockLib.Encryption.Tests/MultiFactory.json

```diff
@@ -1,37 +1,35 @@
 ﻿{
-    "rocklib.encryption": {
-        "CryptoFactories": [
-            {
-                "Type": "RockLib.Encryption.Symmetric.SymmetricCrypto, RockLib.Encryption",
-                "Value": {
-                    "EncryptionSettings": {
-                        "Credentials": [
-                            {
-                                "Name": "Default",
-                                "Algorithm": "Aes",
-                                "IVSize": "16",
-                                "Key": "1J9Og/OaZKWdfdwM6jWMpvlr3q3o7r20xxFDN7TEj6s="
-                            }
-                        ]
-                    }
+    "rocklib.encryption": [
+        {
+            "Type": "RockLib.Encryption.Symmetric.SymmetricCrypto, RockLib.Encryption",
+            "Value": {
+                "EncryptionSettings": {
+                    "Credentials": [
```

Comment:
Don't technically need the square brackets, since it should create a list from a single object

Reply:
Updated.